src/devices/woox.ts

```diff
@@ -86,6 +86,22 @@ const definitions: Definition[] = [
             await reporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0215A', manufacturerName: '_TZ3000_ssp0maqm'}],
```

Comment:
Please add this to the already existing `TS0215A_sos`

Reply:
done, i close this PR i guess ?